lib/dynamo_audits/auditable.rb

```diff
@@ -62,7 +62,8 @@ def write_audit_to_dynamodb(options)
         item: item
       }
 
-      DynamoAudits::AuditJob.perform_later(params)
+      # flatten the write spike
+      DynamoAudits::AuditJob.set(wait: rand(5.minutes)).perform_later(params)
```

Comment:
Is it the right way to set a random delay time?

I have never used this.

Reply:
Looks right